cluster/examples/kubernetes/rook-cluster.yaml

```diff
@@ -30,7 +30,7 @@ spec:
 #    api:
 #      nodeAffinity:
 #      tolerations:
-#    mds:
+#    mgr:
```

Comment:
does the corresponding docs sample need to be updated too? https://github.com/rook/rook/blob/master/Documentation/cluster-crd.md#node-affinity